helpers/tdhttp/test_api.go

```diff
@@ -568,6 +570,38 @@ func (t *TestAPI) CmpHeader(expectedHeader interface{}) *TestAPI {
 	return t
 }
 
+// CmpCookies tests the last request response cookies against
+// expectedCookies. expectedCookies can be a []*http.Cookie or a TestDeep
+// operator. Keep in mind that if it is a []*http.Cookie, it has to match
+// exactly the response cookies. Often only the presence of a
+// cookie key is needed:
+//
+//   ta := tdhttp.NewTestAPI(t, mux).
+//     PostJSON("/login", map[string]string{"name": "Bob", "password": "Sponge"}).
+//     CmdStatus(200).
+//     CmpCookie(td.SuperBagOf(td.Struct(&http.Cookie{Name: "cookie_session"}, nil)))
```

Comment:
Just as a note, could also be:
```suggestion
//     CmpCookie(td.SuperBagOf(td.Struct(&http.Cookie{Name: "cookie_session"}, nil))).
//     CmpCookie(td.SuperBagOf(td.Smuggle("Name", "cookie_session")))
```